### PR TITLE
create schema for registration step and Technical task…

### PR DESCRIPTION
**Pull request by ahmadysalah**
create a schema for registration and technical task in their own files

### CODE REVIEW

**Comment by MohammedYehia** [comment]
@ahmadysalah please keep every branch with its issue.
we agreed that the validation file should be uploaded with the controller file, not the schema

File: server/database/models/registrationStep.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const registrationStep = new Schema({
+  title: { type: String, required: true },
+  details: { type: String, required: true },
```
Comment (Asmaathabet) [inline server/database/models/registrationStep.js]:
```suggestion
title: {
  type: String,
   required: true,
    trim: true,
   },
  details: { 
  type: String, 
  required: true,
  trim: true
  },
```

File: server/database/models/technicalTask.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const technicalTask = new Schema({
+  taskName: { type: String, required: true },
+  taskDescription: { type: String, required: true },
```
Comment (Asmaathabet) [inline server/database/models/technicalTask.js]:
```suggestion
  taskName: {
    type: String,
   required: true,
    trim: true,
    },
  taskDescription: { 
   type: String,
   required: true,
    trim: true,
     },
```

File: server/database/models/technicalTask.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+});
+
+module.exports = mongoose.model('tecTask', technicalTask);
```
Comment (Asmaathabet) [inline server/database/models/technicalTask.js]:
```suggestion
module.exports = mongoose.model('technicalTask', technicalTask);
```

File: server/database/models/registrationStep.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+});
+
+module.exports = mongoose.model('regStep', registrationStep);
```
Comment (Asmaathabet) [inline server/database/models/registrationStep.js]:
```suggestion
module.exports = mongoose.model('registrationStep', registrationStep);
```

Comment (ahmadysalah) [reply server/database/models/registrationStep.js]: i think we can export it by this name

Comment (Asmaathabet) [reply server/database/models/registrationStep.js]: yes but I see this name not suitable, and using abbreviations is not correctly all the time .

Comment (MohammedYehia) [reply server/database/models/registrationStep.js]: @ahmadysalah I think we talked about this and that this is going to be the name of the model

**Pull request merged**